docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,221 @@
+The depolarizing noise model is parameterized by the noise strength, `epsilon`.
+The resulting quasiprobability representations of the `CNOT` gate are then used to obtain an error-mitigated expectation value with Mitiq's
+{ref}`probabilistic error cancellation module <guide/pec/pec>`. 
+For a more in-depth description of the learning function used in this example, see the section on {func}`.learn_depolarizing_noise_parameter`
+in the API-doc.
+The learning-based PEC workflow was inspired by the procedure described in *Strikis et al. PRX Quantum (2021)* {cite}`Strikis_2021_PRXQuantum`.
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+from cirq import (
+    CXPowGate,
+    DepolarizingChannel,
+    Rx,
+    Rz,
+    I,
+    X,
+    Y,
+    Z,
+    LineQubit,
+    Circuit,
+    ops,
+    unitary,
+    InsertStrategy,
+)
+from mitiq import Executor, Observable, PauliString
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.cdr import generate_training_circuits
+from mitiq.cdr._testing import random_x_z_cnot_circuit
+from mitiq.pec import execute_with_pec
+from mitiq.pec.representations.depolarizing import (
+    represent_operation_with_local_depolarizing_noise,
+)
+from mitiq.pec.representations.learning import (
+    biased_noise_loss_function,
+    learn_depolarizing_noise_parameter,
+)
+```
+
+Since the learning-based workflow uses the function {func}`.cdr.generate_training_circuits` from Mitiq's {ref}`Clifford data regression module
+<guide/cdr/cdr>` to generate the near-Clifford training circuits, the input circuit must be one that is compiled into the Rx-Rz-CNOT gateset. 
+Here we use a simple Rx-Rz-CNOT circuit, with an (optional) seed for reproducibility.
+
+```{code-cell} ipython3
+circuit = random_x_z_cnot_circuit(
+    LineQubit.range(2), n_moments=5, random_state=np.random.RandomState(1)
+)
+print(circuit)
+```
+
+
+Define the ideal executor function for simulating the Clifford training circuits without noise. 
+This will be used for comparison to the error-mitigated expectation values in the learning function, and for a final comparison with the
+mitigated and unmitigated values at the end of the workflow.
+
+```{code-cell} ipython3
+def ideal_execute(circ: Circuit) -> np.ndarray:
+    return compute_density_matrix(circ, noise_level=(0.0,))
+
+
+ideal_executor = Executor(ideal_execute)
+```
+
+Define the noisy executor, in this case for simulating depolarizing noise on the `CNOT` gate in each of the training circuits. 
+The optimized value of the noise strength `epsilon` should be close to the value defined for this executor.
+
+```{code-cell} ipython3
+CNOT_ops = list(circuit.findall_operations_with_gate_type(CXPowGate))
+index = CNOT_ops[0][0]
+op = CNOT_ops[0][1]
+operations_to_learn = [Circuit(op)]
+
+epsilon = 0.05
+
+
+def noisy_execute(circ: Circuit) -> np.ndarray:
+    noisy_circ = circ.copy()
+    qubits = op.qubits
+    for q in qubits:
+        noisy_circ.insert(
```

Comment:
This need to change if there are more than 1 CNOTs. 
[Circuit.batch_insert()](https://quantumai.google/reference/python/cirq/Circuit#batch_insert) or https://quantumai.google/reference/python/cirq/map_operations_and_unroll may be useful.

Reply:
thank you, this is what I was looking for in the Cirq docs and concluded didn't exist!